docs/shrc.md

```diff
@@ -22,37 +22,116 @@
 * Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-# Class data sharing
+# Introduction to class data sharing
 
-Sharing class data between JVMs improves start up performance and reduces memory footprint.
+Sharing class data between OpenJ9 VMs improves start up performance and reduces memory footprint. Consider the following outcomes for two VMs that are running similar Java applications but sharing class data:
 
-Start up performance is improved by placing classes that an application needs when initializing into a shared classes cache. The next time the
-application runs, it takes much less time to start because the classes are already available. When you enable class data sharing, AOT compilation is also enabled by default, which dynamically compiles certain methods into AOT code at runtime. By using these features in combination, startup performance can be improved even further because the cached AOT code can be used to quickly enable native code performance for subsequent runs of your application.
+- Start up performance is improved by placing classes that each application needs when initializing into a shared classes cache. The next time the application runs, it takes much less time to start because the classes are already available.
```

Comment:
The performance is actually due to AOT code, JIT hints and profiles.

Reply:
I toned down "much less" to "less". Note that credit is given to AOT improving performance a couple of paragraphs down. I've also added that further improvements are gained by storing JIT data and profiles.